src/redux/home/homeSlice.js

```diff
@@ -0,0 +1,53 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const initialState = {
+  data: [],
+  id: 0,
+  error: '',
+  isLoading: false,
+  search: '',
+};
+
+export const fetchData = createAsyncThunk(
+  'home/fetchdata',
+  async () => {
+    const response = await fetch('https://financialmodelingprep.com/api/v3/symbol/NASDAQ?apikey=JZX3skmwGAztX0ta1ByTbmOON6SAlIax');
```

Comment:
[OPTIONAL] Please add a limit to your API because your API data is too much so your website is loaded very slowly kindly add a limit to increase your website speed 🙂

Reply:
Hello, add the link to the presentation video in the README.md
https://www.loom.com/share/e564c7a4461143608507d40961d828f3?sid=82c53767-6d2e-437a-9f85-734edbaaac4c